tex/A.tex

```diff
@@ -52,24 +52,19 @@
         contract platform. The Argennon blockchain uses
         a hybrid proof of stake (HPoS) consensus protocol, which is capable of combining the benefits of
         a centralized and a decentralized system. In Argennon, ledger storage and transaction processing are
-        outsourced to the cloud and normal personal computers or smartphones, with limited hardware
-        capabilities, can validate transactions and actively
-        participate in the Argennon consensus protocol. This property makes Argennon a truly decentralized and
-        democratic blockchain and one of the most secure existing platforms.
-
-        The Argennon cloud is trustless and publicly verifiable. Computational Integrity (CI) is
-        achieved by using succinct cryptographic proofs (STARK/SNARK)
-        and data integrity is guaranteed by cryptographic accumulators. At the same time, a smart
-        clustering algorithm keeps the bandwidth usage and the overhead of cryptography manageable
+        outsourced to the Argennon cloud while normal personal computers or smartphones, with limited hardware
+        capabilities, can validate transactions and actively participate in the consensus protocol.
+
+        The Argennon cloud consists of a Committee of Delegates (CoD) and Publicly Verifiable Cloud (PVC) servers. 
+        The CoD is elected per term by Proof of Stake and is responsible for processing transactions and creating blocks.
+        After validator assemblies have verified the validity of these blocks they are stored in the PVC servers.
+        PVC servers are publicly verifiable. Computational Integrity (CI) is
+        achieved by using succinct cryptographic proofs (STARK/SNARK) and data integrity is guaranteed by cryptographic accumulators. 
+        At the same time, a smart clustering algorithm keeps the bandwidth usage and the overhead of cryptography manageable
         for validators.
 
-        The Argennon protocol strongly incentivizes the formation of a permission-less network of Publicly Verifiable
```

Comment:
You've totally removed the important fact that the Argennon cloud is permission-less. Any one can create a server and help the Argennon cloud. This makes the cloud decentralized.

Reply:
I thought the incentives were sufficiently explained in detail later on. But yeah we could briefly state it in the introduction already

Reply:
you mean abstract 😁